WordPressKit/PlanServiceRemote_ApiVersion1_3.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+import WordPressShared
+import CocoaLumberjack
+
+@objc public class PlanServiceRemote_ApiVersion1_3: ServiceRemoteWordPressComREST {
+
+    public typealias SitePlans = (activePlan: RemotePlan_ApiVersion1_3, availablePlans: [RemotePlan_ApiVersion1_3])
+
+    @objc public func getPlansForSite(_ siteID: Int,
+                                      success: @escaping (SitePlans) -> Void,
+                                      failure: @escaping (Error) -> Void) {
+        let endpoint = "sites/\(siteID)/plans"
+        let path = self.path(forEndpoint: endpoint, withVersion: ._1_3)
+        let locale = WordPressComLanguageDatabase().deviceLanguage.slug
+        let parameters = ["locale": locale]
+        wordPressComRestApi.GET(
+            path,
+            parameters: parameters as [String : AnyObject]?,
+            success: {
+                response, _ in
+                do {
+                    try success(PlanServiceRemote_ApiVersion1_3.mapPlansResponse(response))
+                } catch {
+                    DDLogError("Error parsing plans response for site \(siteID)")
+                    DDLogError("\(error)")
+                    DDLogDebug("Full response: \(response)")
+                    failure(error)
+                }
+            },
+            failure: {
+                error, _ in
+                failure(error)
+            }
+        )
+    }
+    
+    private static func mapPlansResponse(
```

Comment:
style nitpick: formatting here is weird. i think having the method signature on line line would be cleaner ;)

Reply:
it is a very long line since return type is a tuple, I am updating it as below, how does that look?
    private static func mapPlansResponse(_ response: AnyObject) throws
        -> (activePlan: RemotePlan_ApiVersion1_3, availablePlans: [RemotePlan_ApiVersion1_3]) {